packages/pangea-multipass-llama-index/pyproject.toml

```diff
@@ -1,23 +1,23 @@
-[tool.poetry]
+[project]
 name = "pangea-multipass-llama-index"
 version = "0.2.0"
 description = "Pangea Multipass authorization library for Llama Index"
-authors = ["Apurv Jawle <apurv.jawle@pangea.cloud>"]
+authors = [
+    { name = "Apurv Jawle", email = "apurv.jawle@pangea.cloud" }
+]
 license = "MIT"
 readme = "README.md"
+requires-python = ">=3.10,<4.0"
+dependencies = [
+    "llama-index (==0.12.17)",
```

Comment:
Unrelated to this PR, but is there a reason we pin to a specific llama-index version?

Reply:
Not sure if it's stable enough to allow versions updates without previous checking